worked_examples/geospatial/workshop_april_2024/Workshop_AprilMay2024_extra.ipynb

```diff
@@ -0,0 +1,446 @@
+    "> __**NOTE**__: MapReader can be used to predict the labels on entire datasets and so creating a ``subset_patch_df`` is not needed in most use cases."
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "id": "b12f48db",
+   "metadata": {},
+   "source": [
+    "### Create a dataset (``infer_all``) from our ``patch_df``"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "f482f071",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from mapreader import PatchDataset"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "8757dc4f",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "infer_all = PatchDataset(patch_df, transform=\"val\", patch_paths_col=\"image_path\")"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "id": "94eb2966",
+   "metadata": {},
+   "source": [
+    "### Load dataset into ``my_classifier``"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "b9a1cc46",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "my_classifier.load_dataset(infer_all, \"infer_all\")"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "id": "af9e461f",
+   "metadata": {},
+   "source": [
+    "### Run model inference"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "e3a3f48c",
+   "metadata": {},
+   "source": [
+    "__**TODO**__: Run inference on your ``\"infer_all\"`` dataset"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "f47f6608",
+   "metadata": {
+    "scrolled": false
+   },
+   "outputs": [],
+   "source": [
+    "# my_classifier.inference()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "97c6589b",
+   "metadata": {},
+   "source": [
+    "Save results!"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "2fa7978a",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "my_classifier.save_predictions(\"infer_all\")"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "id": "fd63ec08",
+   "metadata": {},
+   "source": [
+    "### Save results to metadata"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "0baa4808",
+   "metadata": {},
+   "source": [
+    "To add the predictions back into a ``MapImages`` object, we simply need to load our predictions csv file as metadata.\n",
+    "\n",
+    "Since we have started a new notebook, we can create a new ``MapImages`` object by loading our patches."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "85dd41cb",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from mapreader import load_patches"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "438d5700",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "my_maps = load_patches(\n",
+    "    \"./patches_100_meters/png\", parent_paths=\"./maps/*png\"\n",
```

Comment:
This gave me a "no files found" error. I changed to `"./patches_100_meters/*png"`.